src/main.rs

```diff
@@ -1,3 +1,23 @@
-fn main() {
-    println!("Hello, world!");
+pub mod collectors;
```

Comment:
I feel like this compiles the `collectors` module twice, one for the library and one for the binary. We should avoid depending on it here, and instead re-rexport just that function from `monitorbot`.